src/Chapter1.hs

```diff
@@ -652,8 +674,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n < 10 = n
+  | n >= 10 = firstDigit (div n 10)
```

Comment:
Great recursive implementation 💯 

You probably can think of a few ways to make this function work for negative numbers as well 🙂 Otherwise it returns the negative number itself immediately instead of the last digit.